apps/frontend/src/network/teams/__tests__/Outputs.test.tsx

```diff
@@ -1,39 +1,118 @@
+            currentPage: 0,
+            filters: new Set(),
+            pageSize: CARD_VIEW_PAGE_SIZE,
+          }),
+        );
+      }}
+    >
+      <Suspense fallback="loading">
+        <Auth0Provider user={{}}>
+          <WhenReady>
+            <MemoryRouter
+              initialEntries={[
+                {
+                  pathname: network({})
+                    .teams({})
+                    .team({ teamId: '12345' })
+                    .outputs({}).$,
+                },
+              ]}
+            >
+              <Route
+                path={
+                  network({}).teams({}).team({ teamId: '12345' }).outputs({}).$
+                }
+              >
+                <Outputs teamOutputs={teamOutputs} teamId={'12345'} />
+              </Route>
+            </MemoryRouter>
+          </WhenReady>
+        </Auth0Provider>
+      </Suspense>
+    </RecoilRoot>,
+  );
+
+  await waitFor(() =>
+    expect(result.queryByText(/loading/i)).not.toBeInTheDocument(),
   );
+  return result;
+};
+
+it('generates a link to the output (REGRESSION)', async () => {
+  disable('ALGOLIA_RESEARCH_OUTPUTS');
+  const { getByText } = await renderOutputs([
+    { ...createResearchOutputResponse(), id: 'ro0', title: 'Some RO' },
+  ]);
   expect(getByText(/Some RO/).closest('a')!.href).toMatch(/ro0$/);
 });
 
-it('generates a link back to the team', async () => {
-  const { getByText } = render(
-    <StaticRouter location="/team/outputs">
-      <Route path="/team/outputs">
-        <Outputs
-          outputs={[
-            {
-              ...createResearchOutputResponse(),
-              teams: [{ id: '42', displayName: 'Some Team' }],
-            },
-          ]}
-        />
-      </Route>
-    </StaticRouter>,
-  );
+it('generates a link back to the team (REGRESSION)', async () => {
+  disable('ALGOLIA_RESEARCH_OUTPUTS');
+
+  const { getByText } = await renderOutputs([
+    {
+      ...createResearchOutputResponse(),
+      teams: [{ id: '42', displayName: 'Some Team' }],
+    },
+  ]);
   expect(getByText(/Some Team/).closest('a')).toHaveAttribute(
     'href',
     expect.stringMatching(/42$/),
   );
 });
+
+it('renders search and filter', async () => {
+  const { getByRole } = await renderOutputs([
+    { ...createResearchOutputResponse(), id: 'ro0', title: 'Some RO' },
+  ]);
+  expect(
+    (getByRole('searchbox') as HTMLInputElement).placeholder,
+  ).toMatchInlineSnapshot(`"Enter a keyword, method, resource…"`);
+});
+
+it('renders a list of research outputs', async () => {
+  mockGetResearchOutputs.mockResolvedValue({
+    ...createAlgoliaResearchOutputResponse(2),
+    hits: createAlgoliaResearchOutputResponse(2).hits.map((hit, index) => ({
+      ...hit,
+      title: `Test Output ${index}`,
+    })),
+  });
+  const { container } = await renderOutputs([], '');
+  expect(container.textContent).toContain('Test Output 0');
+  expect(container.textContent).toContain('Test Output 1');
+});
```

Comment:
Shouldn't we add tests to cover if it allows typing search queries and applies filters?